sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/internal/changefeed/implementation/LeaseRenewerImpl.java

```diff
@@ -34,66 +34,76 @@ public LeaseRenewerImpl(Lease lease, LeaseManager leaseManager, Duration leaseRe
     public Mono<Void> run(CancellationToken cancellationToken) {
         LeaseRenewerImpl self = this;
 
-        return Mono.fromRunnable( () -> {
-            try {
-                logger.info(String.format("Partition %s: renewer task started.", self.lease.getLeaseToken()));
-                long remainingWork = this.leaseRenewInterval.toMillis() / 2;
+        logger.info(String.format("Partition %s: renewer task started.", self.lease.getLeaseToken()));
```

Comment:
String.format creates unnecessary objects internally. We should avoid it as much as we can. And in this case, I suppose this call can be replaced by something like this: 
`logger.info("Partition {} renewer task started", self.lease.getLeaseToken())`

Reply:
Fixed.